diff -urN b/internal/channelz/BUILD.bazel c/internal/channelz/BUILD.bazel
--- internal/channelz/BUILD.bazel	2000-01-01 00:00:00.000000000 -0000
+++ internal/channelz/BUILD.bazel	2000-01-01 00:00:00.000000000 -0000
@@ -17,6 +17,12 @@
         "//credentials:go_default_library",
         "//grpclog:go_default_library",
     ] + select({
+        "@io_bazel_rules_go//go/platform:darwin": [
+            "@org_golang_x_sys//unix:go_default_library",
+        ],
+        "@io_bazel_rules_go//go/platform:windows": [
+            "@org_golang_x_sys//unix:go_default_library",
+        ],
         "@io_bazel_rules_go//go/platform:android": [
             "@org_golang_x_sys//unix:go_default_library",
         ],
diff -urN a/internal/syscall/BUILD.bazel b/internal/syscall/BUILD.bazel
--- internal/syscall/BUILD.bazel
+++ internal/syscall/BUILD.bazel
@@ -15,12 +15,15 @@ go_library(
         ],
         "@io_bazel_rules_go//go/platform:darwin": [
             "//grpclog:go_default_library",
+            "@org_golang_x_sys//unix:go_default_library",
         ],
         "@io_bazel_rules_go//go/platform:dragonfly": [
             "//grpclog:go_default_library",
+            "@org_golang_x_sys//unix:go_default_library",
         ],
         "@io_bazel_rules_go//go/platform:freebsd": [
             "//grpclog:go_default_library",
+            "@org_golang_x_sys//unix:go_default_library",
         ],
         "@io_bazel_rules_go//go/platform:linux": [
             "//grpclog:go_default_library",
@@ -34,15 +34,18 @@ go_library(
         ],
         "@io_bazel_rules_go//go/platform:netbsd": [
             "//grpclog:go_default_library",
+            "@org_golang_x_sys//unix:go_default_library",
         ],
         "@io_bazel_rules_go//go/platform:openbsd": [
             "//grpclog:go_default_library",
+            "@org_golang_x_sys//unix:go_default_library",
         ],
         "@io_bazel_rules_go//go/platform:plan9": [
             "//grpclog:go_default_library",
         ],
         "@io_bazel_rules_go//go/platform:solaris": [
             "//grpclog:go_default_library",
+            "@org_golang_x_sys//unix:go_default_library",
         ],
         "@io_bazel_rules_go//go/platform:windows": [
             "//grpclog:go_default_library",
